### PR TITLE
Add winners

### PR DESCRIPTION
**Pull request by GresosJ**

### CODE REVIEW

File: src/data/blog.json [hunk 1]
```diff
@@ -1,23 +1,23 @@
 [
   {
-    "author": "",
-    "course": "",
-    "title": "",
-    "summary": "",
-    "text": ""
+    "author": "Carlota da Silva Coelho & Hélder Alexandre Meira Matos",
+    "course": "Filosofia, 1 ano; Ciência Política, 2 ano",
```
Comment (pedrofp4444) [inline src/data/blog.json]:
```suggestion
    "course": "Filosofia, 1º ano & Ciência Política, 2º ano",
```

File: src/data/blog.json [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
+    "link": "https://drive.google.com/file/d/19t536bVmy4PhwNbkeIESBgrq_0Qi6tMx/view?usp=share_link"
   },
   {
-    "author": "",
-    "course": "",
-    "title": "",
-    "summary": "",
-    "text": ""
+    "author": "Helena Rodrigues Antunes",
+    "course": "Direiro, 4º ano",
```
Comment (pedrofp4444) [inline src/data/blog.json]:
```suggestion
    "course": "Direito, 4º ano",
```

File: src/data/blog.json [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
+    "link": "https://drive.google.com/file/d/1LR3yAkmAAek5ddxrpckU-rxqByWiJhF-/view?usp=share_link"
   },
   {
-    "author": "",
-    "course": "",
-    "title": "",
-    "summary": "",
-    "text": ""
+    "author": "Daniel Silva Xavier",
+    "course": "Mestrado em Engenharia Informática, 1º",
```
Comment (pedrofp4444) [inline src/data/blog.json]:
```suggestion
    "course": "Mestrado em Engenharia Informática, 1º ano",
```

**Review by pedrofp4444 — CHANGES_REQUESTED** [review]

**Review by pedrofp4444 — APPROVED** [review]
Great work :muscle:

**Comment by MarioRodrigues10** [comment]
@GresosJ , in order to say everything in the same context, we need permitions to see Helena's paper can you fix that?
Thanks :pray:

**Pull request merged**